src/FINDOLOGIC/Api/ResponseObjects/Xml/XmlResponse.php

```diff
@@ -48,11 +46,11 @@ class XmlResponse
     private $filterAmount = 0;
 
     /**
-     * XmlResponse constructor.
-     * @param $response
+     * @inheritdoc
      */
-    public function __construct($response)
+    public function __construct($response, $responseTime = null)
     {
+        $this->responseTime = $responseTime;
         $xmlResponse = new SimpleXMLElement($response);
```

Comment:
SDYM support seems to be missing.

Reply:
How do you mean that? You can get everything from the `ResponseObjects\Xml\Properties\Query`. There will be helpers implemented in a later version, but that is something that is already described with more detail in #29.

Reply:
Overlooked it, sorry.